verification/curator-service/ui/src/components/App/App.test.tsx

```diff
@@ -110,7 +114,7 @@ describe('<App />', () => {
 
         expect(await screen.findByTestId('mapLink')).toHaveAttribute(
             'href',
-            'https://map.covid-19.global.health/',
+            'http://dev-map.covid-19.global.health/',
```

Comment:
https?

Reply:
Dev version of the map is served over http and not https